addon/components/cfb-form-editor/question/options.js

```diff
@@ -83,6 +101,52 @@ export default RenderComponent.extend({
     this.setDirty();
   },
 
+  setupUIkit() {
+    UIkit.util.on("#option-list", "moved", (...args) =>
+      run(this, this._handleMoved, ...args)
+    );
+  },
+
+  _handleMoved({ detail: [sortable] }) {
+    const options = [...sortable.$el.children];
+
+    this.reorderQuestions.perform(
+      options.map((option) =>
+        addQuestionPrefix(option.firstElementChild.id, this.questionSlug)
+      )
+    );
+  },
+
+  reorderQuestions: task(function* (slugs) {
+    try {
+      slugs.pop();
```

Comment:
Why this? Also, I'd prefer a functional approach like `const [ slugsWithoutLast, _ ] = slugs;`. A comment why this is done would be nice as well

Reply:
Moved it and added a comment.
The suggestion you wrote doesnt work as it would only take the first 2 elements from the array and you cant reverse the rest operator.